ably.d.ts

```diff
@@ -2332,34 +2332,40 @@ export interface Message {
   name?: string;
   /**
    * Timestamp of when the message was received by Ably, as milliseconds since the Unix epoch.
+   * (This is the timestamp of the current version of the message)
```

Comment:
The docstrings for all of these new fields should have been added to https://github.com/ably/sdk-api-reference at the same time as they were added to the spec (that repo should be the source of all docstrings that get added to our SDKs); please could you add them there?

Reply:
@coderabbitai create a task to add mentioned requirement to the [CONTRIBUTING.md](https://github.com/ably/ably-js/blob/main/CONTRIBUTING.md)

Reply:
please try again, but don't create it in jira, create an issue in this github repo

Reply:
created https://github.com/ably/ably-js/issues/1929 to clarify about docstrings in [CONTRIBUTING.md](https://github.com/ably/ably-js/blob/main/CONTRIBUTING.md)

Reply:
https://github.com/ably/sdk-api-reference/pull/44

Reply:
I think that should be merged into the spec repo. I don't see why there's a separate repo just for docstrings.